### DISCUSSION TITLE
ecore-glsp status

### DISCUSSION THREAD

**Discussion by BeniBianchi** [eclipse-emfcloud/emfcloud#218] [All other]
Hi,

the ecore-glsp example seems to be very interesting.

Are there any plans to update it to the latest versions of Theia, GLSP, and EMF.cloud?

Unfortunately I don't have enough experience with these frameworks to do it myself.

Best regards,
Beni

**Comment by ndoschek** [comment]
Hi @BeniBianchi,

The @eclipse-emfcloud/ecore-glsp repository contains a demonstrator for showing an editor for Ecore models. Thus, we only focused on what’s relevant for the demonstration under a controlled environment. Unfortunately there are no plans on our end to update the demonstrator currently.
If you are interested in transforming this demonstrator into an adoptable state, e.g. by funding the development work or working on it yourself, please let us know! :)

Besides that, the [`coffee-editor`](https://github.com/eclipse-emfcloud/coffee-editor) was updated more recently, but also has additional features. The used base frameworks are the same however in case you'd like to have a look at more updated code base.

HTH and best regards,
Nina
